code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -535,6 +539,8 @@ GLOBAL_DATUM_INIT(sdql2_vv_statobj, /obj/effect/statclick/SDQL2_VV_all, new(null
 					objs[j] = SDQL_expression(x, expression)
 					SDQL2_TICK_CHECK
 					SDQL2_HALT_CHECK
+				if(length(objs) == 1 && islist(objs[1]))
```

Comment:
What's with conditional unwrapping, this is going to be an issue if anyone wants to retrieve list variables. This should probably be controlled by the user.

Reply:
This is to get around an issue where defining a list then immediately accessing it doesn't work. I'll see what I can do about reverting this and fixing the issue itself.

Reply:
I've reverted the unwrapping and allowed variable to process expressions now. The proper solution to my above problem is `(@[/area/crew_quarters/bar MAP contents])[1]`